src/checks/stylelint.ts

```diff
@@ -0,0 +1,40 @@
+import { execaCommand } from 'execa';
+
+export const title = 'Stylelint';
+
+export default async function stylelintCheck() {
+  try {
+    await execaCommand(
+      'pnpm stylelint **/*.{css,scss,less,js,tsx} --max-warnings 0',
```

Comment:
did we also want to support `.jsx` here?

Reply:
58f22fa

Reply:
https://github.com/upleveled/preflight/pull/377/commits/58f22fa912f16a6bb5ae8e43bc338b72ed424199